build-package.sh

```diff
@@ -1,6 +1,13 @@
 #!/bin/bash
 # shellcheck disable=SC1117
 
+# Fixing the number of calls and the list of compiled packages
+if [[ ! "$TERMUX_BUILD_PACKAGE_CALL_DEPTH" =~ ^[0-9]+$ ]]; then
+	export TERMUX_BUILD_PACKAGE_CALL_DEPTH=0
+else
+	export TERMUX_BUILD_PACKAGE_CALL_DEPTH=$((TERMUX_BUILD_PACKAGE_CALL_DEPTH+1))
```

Comment:
Should combine this `if` with the one below, the variables are not needed before it.

Reply:
I'm sorry, but again I didn't understand you. What do you mean by "the one below"?

Reply:
```shell
if [[ ! "$TERMUX_BUILD_PACKAGE_CALL_DEPTH" =~ ^[0-9]+$ ]]; then
	export TERMUX_BUILD_PACKAGE_CALL_DEPTH=0
	export TERMUX_BUILD_PACKAGE_CALL_BUILT_PACKAGES_LIST_FILE_PATH="${TMPDIR}/build-package-call-built-packages-list-$(date +"%Y-%m-%d-%H.%M.%S.")$((RANDOM%1000))"
	echo -n " " > "$TERMUX_BUILD_PACKAGE_CALL_BUILT_PACKAGES_LIST_FILE_PATH"
else
	export TERMUX_BUILD_PACKAGE_CALL_DEPTH=$((TERMUX_BUILD_PACKAGE_CALL_DEPTH+1))
fi
```

Reply:
thank